File: java/03_007.java
public class Fibonacci {
    public static int fibonacci(int n) {  // O(2ⁿ)
        if (n <= 1) {
            return n;
        }
        return fibonacci(n-1) + fibonacci(n-2);
    }
    
    public static void main(String[] args) {
        // Example usage
        int n = 10;
        System.out.println("Fibonacci(" + n + ") = " + fibonacci(n));
    }
}

File: data structures and algorithms/cs/03_007.cs
public class Fibonacci
{
   public static int Fibonacci(int n)
   {
      if (n <= 1)
      {
         return n;
      }
      return Fibonacci(n - 1) + Fibonacci(n - 2);
   }

   public static void Main(string[] args)
   {
      // Example usage
      int n = 10;
      System.Console.WriteLine(
         "Fibonacci(" + n + ") = " + Fibonacci(n));
   }
}